File error handling (#30)
prestwick/systemlink-migration-sandbox: README.md

### Basic usage

```bash
py systemlinkmigrate.py capture --tags
```
Running `systemlinkmigrate.py` with the above arguments will capture tag and tag history and store this data in `C:\migration`. 

```bash
py systemlinkmigrate.py restore --tags
```

Running `systemlinkmigrate.py` with the above arguments will restore tag and tag history data from the directory `C:\migration`.

### Capture and Restore

> The `capture` and `restore` actions determine the directionality of the migration. The `capture` action is used when migrating data FROM an existing SystemLink server. The `restor` action is used when migrating data TO a new SystemLink server. Both actions cannot be used simultaneously. 

Reviewer: spelling. `restor`

## Migrating tag history to nitaghistorian database
Due to a bug introduced in SystemLink 2020R2 it is possible for tag history data to be stored in the incorrect database within the MongoDB instance. **This bug only affects environments where a remote MongoDB server is used.** To determine if you are in this state connect a MongoShell or visual database tool such as [Mongo Compass](https://www.mongodb.com/products/compass) to your Mongo instance and check if the collections `metadata` and `values` are in the default authentication database (typically the `admin` database).

> - Use the `thdbbug` action to correct this. No other migration actions may be paired with this argument. 

Reviewer 2: ```suggestion
- Use the `thdbbug` action to correct this. Cannot be used simultaneously with other actions. 
```
